CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to the Amplitude SDK for Node
+
+🎉 Thanks for your interest in contributing! 🎉
+
+## Ramping Up
+
```

Comment:
any thoughts on this? i feel like i could be better about marking issues as `good first issue`

Reply:
Something we should definitely expand in the future